prediction-polls/android/app/src/main/java/com/bounswe/predictionpolls/data/remote/TokenManager.kt

```diff
@@ -15,11 +19,31 @@ class TokenManager @Inject constructor(@ApplicationContext context: Context) {
 
     private val prefs = context.getSharedPreferences(PREFERENCE_FILE_KEY, Context.MODE_PRIVATE)
 
+    private val _accessTokenFlow = MutableStateFlow(prefs.getString(ACCESS_TOKEN_KEY, null))
+    val accessTokenFlow: Flow<String?> = _accessTokenFlow.asStateFlow()
+
+    private val _refreshTokenFlow = MutableStateFlow(prefs.getString(REFRESH_TOKEN_KEY, null))
+    val refreshTokenFlow: Flow<String?> = _refreshTokenFlow.asStateFlow()
```

Comment:
casting mutable state flow to state flow and exposing it as a flow seems sus to me

Reply:
that was only thing come up to my mind. since it was working i left it.